TheWarWithin/Priorities/RogueOutlaw.simc

```diff
@@ -65,14 +73,15 @@ actions.cds+=/roll_the_bones,if=set_bonus.tww2_4pc&rtb_buffs.will_lose<=1&(varia
 actions.cds+=/roll_the_bones,if=set_bonus.tww2_4pc&(rtb_buffs<=2|(rtb_buffs.max_remains<11|!talent.keep_it_rolling)&rtb_buffs.will_lose<5&talent.supercharger)
 # Without TWW2 set or Sleight of Hand, recast Roll the Bones to override 1 buff into 2 buffs with Loaded Dice, or reroll any 2 buffs with Loaded Dice+Supercharger. Hidden Opportunity builds can also reroll 2 buffs with Loaded Dice to try for BS/RP/TB.
 actions.cds+=/roll_the_bones,if=!set_bonus.tww2_4pc&(rtb_buffs.will_lose<=buff.loaded_dice.up|talent.supercharger&buff.loaded_dice.up&rtb_buffs<=2|talent.hidden_opportunity&buff.loaded_dice.up&rtb_buffs<=2&!buff.broadside.up&!buff.ruthless_precision.up&!buff.true_bearing.up)
-actions.cds+=/ghostly_strike,if=combo_points<cp_max_spend
 # Trinkets that should not be used during stealth and have higher priority than entering stealth.
 actions.cds+=/use_item,name=imperfect_ascendancy_serum,if=!stealthed.all|boss&fight_remains<=22
 actions.cds+=/use_item,name=mad_queens_mandate,if=!stealthed.all|boss&fight_remains<=5
-# Killing Spree has higher priority than entering stealth.
-actions.cds+=/killing_spree,if=variable.finish_condition&!stealthed.all
-# Builds with Crackshot, Underhanded Upper Hand and Subterfuge use Vanish while Adrenaline Rush is active, the finisher condition is met, and not already in stealth. Trickster builds also consume Coup de Grace before Vanishing if it is ready.
-actions.cds+=/call_action_list,name=vanish_usage,if=!stealthed.all&talent.crackshot&talent.underhanded_upper_hand&talent.subterfuge&buff.escalating_blade.stack<4&buff.adrenaline_rush.up&variable.finish_condition
+# If necessary, standard builds prioritize using Vanish at any CP to prevent Adrenaline Rush downtime.
```

Comment:
What.

This one may end up needing a `group&` conditional so that we don't get mass reports about dropping combat when people are soloing.  Then again, there's already a spec option for that.